common/lib/capa/capa/responsetypes.py

```diff
@@ -380,7 +380,9 @@ def make_hint_div(self, hint_node, correct, student_answer, question_tag,
             style = QUESTION_HINT_INCORRECT_STYLE
 
         # Ready to go
-        return u'<div class="{0}">{1}{2}</div>'.format(style, label_wrap, hints_wrap)
+        # pylint: disable=line-too-long
```

Comment:
Get rid of this comment and correct the line length. You should be able to use parenthesis to achieve this.